content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -119,11 +134,11 @@ When variables from [committed `.auto.tfvars` files](#loading-variables-from-fil
 
 Consider an example workspace that has the following variables applied:
 
-| Name | Scope | Date Applied | ACCESS_KEY | ACCESS_ID | VAR1 | KEY1 | VAR2|
-|-----|-------|-------|-----|-------|-------|--------|--------|------|
-| Variables | Workspace-Specific | 10/1 | `g47fh474` | `874hf7u4` | `h` |     |     |
-| Variable Set A | Non-Global | 10/4 |   |   | `y` | `x` |   |   |
-| Variable Set B | Global | 10/20 |   |     |   |  `z` | `a` |
+| Name           | Scope              | Date Applied | ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 |
```

Comment:
I had originally excluded it because I would need to add a new example variable to the table, but I think it's a good tradeoff. Let me know what you think.